frontend/views/containers/contributions/SupportHistory.vue

```diff
@@ -41,32 +39,18 @@ export default ({
   computed: {
     ...mapGetters([
       'currentPaymentPeriod',
-      'periodStampGivenDate',
-      'periodBeforePeriod',
       'withGroupCurrency',
       'groupTotalPledgeAmount',
       'groupCreatedDate'
-    ]),
-    firstDistributionPeriod () {
-      // group's first distribution period
-      return this.periodStampGivenDate(this.groupCreatedDate)
-    },
-    periods () {
-      const periods = [this.currentPaymentPeriod]
-      for (let i = 0; i < MAX_HISTORY_PERIODS - 1; i++) {
-        const period = this.periodBeforePeriod(periods[0])
-        if (comparePeriodStamps(period, this.firstDistributionPeriod) < 0) break
-        else periods.unshift(period)
-      }
-      return periods
-    }
+    ])
   },
   mounted () {
     this.updateHistory()
   },
   methods: {
     async updateHistory () {
-      this.history = await Promise.all(this.periods.map(async (period, i) => {
+      const periods = await this.getSortedPeriodKeys()
```

Comment:
`this.getSortedPeriodKeys` doesn't exist!

(**Please** test code before pushing to make sure it works. I might not always be good of a reviewer to catch stuff like this, but the computer will always catch it if you test these cases)

Reply:
Well I ran our test suite - I'm unfortunately no better at catching typos or old names, and need tools to do it for me